tests/UI/pages/BO/catalog/products/index.js

```diff
@@ -429,6 +478,27 @@ class Product extends BOBasePage {
     return this.getTextContent(page, this.alertSuccessBlockParagraph);
   }
 
+  /**
+   * Bulk set status
+   * @param page
+   * @param status
+   * @return {Promise<string>}
+   */
+  async bulkSetStatus(page, status) {
+    await Promise.all([
+      this.waitForVisibleSelector(page, this.productBulkMenuButton),
+      page.click(this.selectAllBulkCheckboxLabel),
+    ]);
+
+    await Promise.all([
+      this.waitForVisibleSelector(page, `${this.productBulkMenuButton}[aria-expanded='true']`),
```

Comment:
Can't we make this selector an attribute ?

Reply:
done